AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/related.js

```diff
@@ -109,8 +109,10 @@ define([
             }).done(function (data) {
                 var relatedImages = this.relatedImages();
 
-                relatedImages.series[record.id] = data.result['same_series'];
-                relatedImages.model[record.id] = data.result['same_model'];
+                relatedImages.series[record.id] = data.result['same_series'].length > 0 ?
```

Comment:
I think it would be better to introduce `isTabVisible` method and merge the logic introduced here and in the template as part of that method: `function (record, tabName) { return typeof this.relatedImages()[tabName][record.id] !== 'undefined' && data.result['same_series'].length > 0 }`